examples/advanced/custom-filters/config.js

```diff
@@ -0,0 +1,115 @@
+const StyleDictionary = require("style-dictionary");
+const tokens = require("./properties");
+
+module.exports = {
+  source: ["properties/**/*.json"],
+  platforms: {
+    "esm/category": {
+      transformGroup: "js",
```

Comment:
It looks like you are defining both `transforms` and a `transformGroup`. I think the `transforms` will override anything in `transformGroup` so we might be able to remove this line (and the other transformGroup lines below)